src/widgets/search-box/search-box.js

```diff
@@ -173,8 +207,9 @@ searchBox({
  * @property  {string|HTMLElement} container CSS Selector or HTMLElement to insert the widget.
  * @property  {string} [placeholder] Input's placeholder.
  * @property  {boolean|SearchBoxPoweredByOption} [poweredBy=false] Define if a "powered by Algolia" link should be added near the input.
- * @property  {boolean|SearchBoxResetOption} [reset=true] Define if a reset button should be added in the input when there is a query.
+ * @property  {boolean|SearchBoxResetOption} [reset=false] Define if a reset button should be added in the input when there is a query.
  * @property  {boolean|SearchBoxMagnifierOption} [magnifier=true] Define if a magnifier should be added at beginning of the input to indicate a search input.
+ * @property  {boolean|SearchBoxLoadingIndicatorOption} [loadingIndicator=false] Define if a loading indicator should be added at beginning of the input to indicate that search is currently stalled.
```

Comment:
Should we make it true by default, since it's a new feature and might bring good behaviour by default?

Reply:
I was wondering. The idea behind `false` by default is that we don't change any existing behavior / style, especially if the user doesn't use the provided stylesheets.

Reply:
Yep that's the safest, if we were to do a new breaking version we would make it true by default, cc @mthuret can we add that to the feedback/product loop?

Reply:
Happy to break in v3 ;)

Reply:
agreed, false for now, breaking to `true` in v3 👍